build.gradle

```diff
@@ -27,12 +27,19 @@ dependencies {
     implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.10'
     implementation group: 'org.zalando', name: 'problem', version: '0.23.0'
     implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: '0.23.0'
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit5Version
+
+    // map ordinals to numbers
+    implementation group: 'com.ibm.icu', name: 'icu4j', version: '66.1'
+
+    //Used for different testing utilities such as reading from resources
+    testImplementation group:'com.github.BIBSYSDEV', name:'aws-build-tools', version:'v1.5.5'
+
     testImplementation group: 'org.mockito', name: 'mockito-core', version: '3.2.4'
     testImplementation group: 'com.github.stefanbirkner', name: 'system-rules', version: '1.19.0'
-    testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit5Version
-    testCompileOnly 'junit:junit:4.12'
-    testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.1.0'
+    testImplementation group: 'org.junit.jupiter', name:'junit-jupiter', version:"${junit5Version}"
```

Comment:
You can refer to the defined variable directly, without concatenation

Reply:
done